$OpenBSD: patch-kde-dev-scripts_qt4_adapt-kconfiggroupsaver_pl,v 1.2 2013/07/05 14:32:50 zhuk Exp $
Cleanup development scripts, making them more portable and secure.
Upstream review 110858: http://git.reviewboard.kde.org/r/110858/
--- kde-dev-scripts/qt4/adapt-kconfiggroupsaver.pl.orig	Fri Jun 28 22:03:13 2013
+++ kde-dev-scripts/qt4/adapt-kconfiggroupsaver.pl	Fri Jul  5 18:14:32 2013
@@ -9,7 +9,7 @@ use functionUtilkde;
 use strict;
 
 
-open(my $F, q(find -name "*" |));
+open(my $F, "-|", qw(find . -type f));
 my $file;
 while ($file = <$F>) {
     chomp $file;
@@ -78,7 +78,7 @@ while ($file = <$F>) {
 	    } <$FILE>;
 
 	if ($modified) {
-	    open (my $OUT, ">$file");
+	    open (my $OUT, ">", $file);
 	    print $OUT @l;
 	}
 	if ($modified )
